src/main/java/redis/clients/jedis/MultiKeyPipelineBase.java

```diff
@@ -8,7 +8,7 @@
 
 public abstract class MultiKeyPipelineBase extends PipelineBase implements
     MultiKeyBinaryRedisPipeline, MultiKeyCommandsPipeline, ClusterPipeline,
-    BinaryScriptingCommandsPipeline, ScriptingCommandsPipeline {
+    BinaryScriptingCommandsPipeline, ScriptingCommandsPipeline, BasicRedisPipeline {
```

Comment:
We don't need to add this since PipelineBase should implement BasicRedisPipeline.